projects/ngx-maplibre-gl/src/lib/source/geojson/geojson-source.component.ts

```diff
@@ -244,4 +244,40 @@ export class GeoJSONSourceComponent implements OnChanges {
       })
     );
   }
+
+  /**
+   * Sets the `state` of a feature.
```

Comment:
This is optional, but you might be able to define an inteface for the sources (instead of a base class) and define these interface methods with the docs and use `@inheritDoc` to avoid the need to write these comments over and over again.
This is optional.